packages/client/src/v2-events/features/events/registered-fields/Address.stories.tsx

```diff
@@ -0,0 +1,271 @@
+/*
```

Comment:
Restructured the stories so that Address story demonstrates Input, Output and "toString" capabilities